flink/src/main/java/org/apache/sedona/flink/expressions/Predicates.java

```diff
@@ -104,4 +104,28 @@ public Boolean eval(@DataTypeHint("INT") Integer key, @DataTypeHint(value = "RAW
             return JudgementHelper.match(geom1, geom2, halfOpenRectangle, false);
         }
     }
+
+    public static class ST_Disjoint extends ScalarFunction {
+        private List<Envelope> grids;
+
+        /**
+         * Constructor for duplicate removal
+         */
+        public ST_Disjoint(PartitioningUtils partitioner) {
+            grids = partitioner.fetchLeafZones();
+        }
+
+        /**
+         * Constructor for relation checking without duplicate removal
+         */
+        public ST_Disjoint() {
+        }
+
+        @DataTypeHint("Boolean")
+        public Boolean eval(@DataTypeHint(value = "RAW", bridgedTo = org.locationtech.jts.geom.Geometry.class) Object o1, @DataTypeHint(value = "RAW", bridgedTo = org.locationtech.jts.geom.Geometry.class) Object o2) {
+            Geometry geom1 = (Geometry) o1;
```

Comment:
Please also add a test to Flink. Thanks!

Reply:
@jiayuasu is there a command to only run flink tests?

Reply:
In Intelij IDEA, go to the test case you want to run. Right click the test case in the source code to run the specific test case